models/pathology_nuclei_classification/configs/inference.json

```diff
@@ -0,0 +1,116 @@
+{
+    "imports": [
+        "$import glob",
+        "$import json",
+        "$import pathlib",
+        "$import os"
+    ],
+    "bundle_root": "/workspace/data/pathology_nuclei_classification",
```

Comment:
Hi @SachidanandAlle , here could the bundle root be changed as[ this ticket](https://github.com/Project-MONAI/model-zoo/issues/247) suggest? We planned to standardize the way to define `bundle_root`, thanks!

Reply:
same for `train.json` and the other bundle.

Reply:
on `bundle_root` i will update over another PR once we arrive a conclusion regarding the same. hope that's not a blocker to merge the PR